### PR TITLE
#5 Info and Relationships What's Next to be Loaded? Table - Based on master

### PR DESCRIPTION
**Pull request by michellemcgowan**
Address issue #5. Table cells were not associated to the horizontal headers. 

<h2>Code Changes</h2>

- formatted file
- added `scope="col"` for the table vertical headers
- added `scope="row"` for the table horizontal headers or the queue numbers
- table styling 
  - looks better now
  - added extra row so last row w/ info has a line underneath it
  - added a `<td>` to each row so that the line goes all the way to comments even if it doesn't have one 
- added `<h5>` heading "Table 1: Load Data Queue" above the table to be consistent w/ other pages and allow for accessibility 

image
image

### CODE REVIEW

File: WhatsNext.html [hunk 1]
```diff
@@ -86,98 +96,90 @@ <h3>The Queue</h3>
         The table below details our current plan for loading data to PROD. This table includes the state and
         date we plan to load the data.
       </p>
+      <h5 id="loadDataQueue"> Table 1: Load Data Queue</h5>
```
Comment (Kelvyn-Meyers) [inline WhatsNext.html:99]: No change required here, but I recommend merging in master one more time now that new h5 CSS has been added. We can see how it looks with your updated table.

**Review by Kelvyn-Meyers — CHANGES_REQUESTED** [review]
Hi @michellemcgowan,

Good work with making the table more accessible and improving its visuals! I just have a few notes regarding some potential Parsing errors that were introduced and a suggestion for blank comments. See my in-line comments below and let me know if you have any questions or better ideas!

File: WhatsNext.html [hunk 2]
```diff
@@ -86,98 +96,90 @@ <h3>The Queue</h3>
...
+      <th scope="row">       
+      </th>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <p>
+    </p>
 </div>
```
[the remark this answers is not in the document]
Comment (michellemcgowan) [reply WhatsNext.html]: Got it! 
- added a bottom-border to the last tr
- there was also no closing tag on the other page for the last table so i added both of those
- removed `<p>` tags

File: WhatsNext.html (hunk 1 above)
Comment (michellemcgowan) [reply WhatsNext.html:99]: image

**Review by Kelvyn-Meyers — APPROVED** [review]
Great work here @michellemcgowan! This page should now fully pass Info & Relationships. I added one commit which makes the bottom-border for tables apply dynamically to the last `<tr>` in each table to avoid needing to change this manually. Also, good catch on the `</table>` tag for the Loaded Data page. We will definitely need to cleanup all of these pages once we get to Parsing tickets. Ready to merge!

**Pull request merged**